src/main/java/DilemmaDetector/Consequences/ConsequenceContainer.java

```diff
@@ -0,0 +1,97 @@
+package DilemmaDetector.Consequences;
+
+import generator.ObjectNamer;
+import project.*;
+
+import java.util.*;
+
+public class ConsequenceContainer implements IConsequenceContainer {
+    private Map<String, Map<ConsequenceType, Set<String>>> healthConsequences = new HashMap<>(); //<Decision, <Consequence, Set<VictimName>>>
```

Comment:
Nazwa zmiennej wskazuje na coś bardzo prostego, a dostajemy dosyć złożony typ.

1. Można trochę uprościć typ poprzez zrobienie typu dla Map<ConsequenceType, Set>, ale niekoniecznie to będzie zrozumiałe
2. Dłuższa nazwa healthConsequencesByDecisionNames